testing/discovery_interface.py

```diff
@@ -39,12 +49,21 @@ def check_discovery_status():
     return True if r and r.status_code == 200 else False
 
 
+def discovery_container_is_running():
+    """
+    Checks whether the discovery container is still running
+    """
+    if subprocess.check_output("docker ps -aq -f status=exited -f name={}".format(CONTAINER_NAME), shell=True):
+        return False
+    return True
+
+
 def try_discovery(attempt_number):
     try:
         check_discovery_status()
         return True
     except Exception:
-        if attempt_number >= 3:
+        if attempt_number >= 15:
```

Comment:
should this `15` also be set similarly to `RETRIES` and `TIMEOUT` in `testing/discovery_config.py`?

Reply:
Addressed in https://github.com/finos/greenkey-discovery-sdk/pull/131/commits/208a3a08a1816022b69b7c9e9f216b4b4e6ca1d3